src/ReachSets/DiscretePost/ConcreteDiscretePost.jl

```diff
@@ -1,34 +1,34 @@
-# ====================================================
-# Textbook implementation of a discrete post operator.
-# Uses concrete intersection.
-#
+"""
+    ConcreteDiscretePost <: DiscretePost
+
+Textbook implementation of a discrete post operator, using concrete polyhedra
+intersections.
+
 ### Notes
-#
-# The current implementation requires that the `Polyhedra` # library is loaded,
-# because some concrete operations between polytopes are used.
-#
-# Currently, we assume that source invariants, target invariants and guards are
-# polytopes in constraint representation resp. HalfSpaces.
-#
+
+This operator requires that the `Polyhedra` library is loaded,
+because some concrete operations between polytopes are used.
+
+Currently, we assume that source invariants, target invariants and guards are
+polytopes in constraint representation resp. half-spaces.
+
 ### Algorithm
```

Comment:
I am not sure if it makes sense to write about an algorithm for a type. I know what is meant here, but it can be confusing.

Reply:
Hmm point taken. I've created #378.